tests/services/schemas/test_related_identifier.py

```diff
@@ -159,6 +161,7 @@ def test_invalid_related_identifiers(app, minimal_record):
         data = MetadataSchema().load(metadata)
 
 
+@pytest.mark.skip()
```

Comment:
Question: Why the skip?

Reply:
Discussed IRL, deduplication is:
- Incorrect the way it was before. i.e. there can be more than one related DOI work...
- An overkill to do a complex deduplication validation